scripts/cram

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# if we have cram already, use that
```

Comment:
would rather have it just use the builtin cram... "principle of least surprise"... but I don't think it's a problem